tools/fix_auth.rb

```diff
@@ -13,6 +13,7 @@
 
 require 'active_support/all'
 require 'active_support/concern'
+require 'manageiq-gems-pending'
```

Comment:
@kbrock Sorry, forgot to ping you on this question... I'm just trying to understand why this didn't fail the tests and I'm assuming it's because a prior test already loaded this code.